web/workflow.R

```diff
@@ -39,29 +39,36 @@ if (is.na(args[1])){
   settings <- read.settings(settings.file)
 }
 
+# Update/fix/check settings. Will only run the first time it's called, unless force=TRUE
+settings <- fix.deprecated.settings(settings, force=FALSE)
+settings <- addSecrets(settings, force=FALSE)
+settings <- update.settings(settings, force=FALSE)
+settings <- check.settings(settings, force=FALSE)
+
 # Check for additional modules that will require adding settings
 if("benchmark" %in% names(settings)){
   library(PEcAn.benchmark)
   settings <- papply(settings, read.settings.RR)
 }
 
 # Write pecan.CHECKED.xml
-settings <- write.settings(settings, outputfile = "pecan.CHECKED.xml")
+write.settings(settings, outputfile = "pecan.CHECKED.xml")
 
 # start from scratch if no continue is passed in
-if (length(which(commandArgs() == "--continue")) == 0) {
-  file.remove(file.path(settings$outdir, "STATUS"))
+statusFile <- file.path(settings$outdir, "STATUS")
+if (length(which(commandArgs() == "--continue")) == 0 && file.exists(statusFile)) {
+  file.remove(statusFile)
 }
   
 # Do conversions
-settings <- do.conversions(settings)
+settings <- do.conversions(settings, overwrite.met=FALSE, overwrite.fia=FALSE)
```

Comment:
These are the default values (which themselves are hack) so I'm not sure we need to expose them at this time. Probably better to wait until we generalize this syntax.